examples/tutorials/lighting_tutorial.py

```diff
@@ -100,7 +100,7 @@ def main(show_imgs=True, save_imgs=False):
 
```

Comment:
I've made some minimal updates to the lighting tutorial, but I'd like to see us redo this in the future to properly explain lighting and material concepts. We can probably do this to coincide with PBR coming online. (Related, a pet peeve of mine: we shouldn't be demonstrating dynamic lighting applied to the van gogh room, because that model has baked lighting.)